upgrades/steps_27.go

```diff
@@ -79,14 +89,97 @@ func stepsFor27() []Step {
 	}
 }
 
+func setLogOwnerCorrectLogPermissions(filename string) error {
+	group, err := user.LookupGroup("adm")
```

Comment:
I gave this function as an outline. As you have already identified, centos and suse have different groups. We should perhaps make the function take owner and group names.

Reply:
I just put it up only, actually was not ready to be reviewed, wanted to save over the weekend. Sorry :slightly_frowning_face: